spec/model/idea_spec.rb

```diff
@@ -1,15 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Idea, type: :model do
```

Comment:
Those generators get me some times. I think what happened here was I had a duplicate spec and one had info the other didn't so I just put it all together